app/src/main/java/com/conupods/IndoorMaps/IndoorBuildingOverlays.java

```diff
@@ -0,0 +1,232 @@
+package com.conupods.IndoorMaps;
+
+import android.view.View;
+
+import com.conupods.IndoorMaps.ConcreteBuildings.CCBuilding;
+import com.conupods.IndoorMaps.ConcreteBuildings.HBuilding;
+import com.conupods.IndoorMaps.ConcreteBuildings.MBBuilding;
+import com.conupods.IndoorMaps.ConcreteBuildings.VLBuilding;
+import com.conupods.OutdoorMaps.Building;
+import com.conupods.R;
+import com.google.android.gms.maps.GoogleMap;
+import com.google.android.gms.maps.model.BitmapDescriptor;
+import com.google.android.gms.maps.model.BitmapDescriptorFactory;
+import com.google.android.gms.maps.model.GroundOverlay;
+import com.google.android.gms.maps.model.GroundOverlayOptions;
+import com.google.android.gms.maps.model.LatLng;
+import com.google.android.gms.maps.model.MapStyleOptions;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class IndoorBuildingOverlays {
+
+    private GoogleMap mMap;
+
+    public enum Buildings{
```

Comment:
should rename to BuildingCodes

Reply:
and hall should be consistent with the building code in json file

Reply:
oh very true! changed it to H, and renamed enum